### PR TITLE
added interview of Aisuko Li

### PR DESCRIPTION
**Pull request by hargunkaur286**
**Description**

This PR fixes #
@vishalvivekm I was not sure of:
1. what date to add for the interview, so I added a random date. Please let me know if I need to change that. 
2. Link to the adapters project

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit be7857f49b76fb229eb23be42387c75046d44d0b at: https://6710cc8006ee75bc43db7465--layer5.netlify.app

**Comment by Touriist** [comment]
@hargunkaur286 clicking on Anita's picture takes to community/members/undefined
image

**Comment by vishalvivekm** [comment]
@hargunkaur286 
Thank you for your contribution!
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by Aisuko** [comment]
What should I do for this PR? I think maybe this link can be as a reference https://layer5.io/blog/announcements/announcing-meshery-v050

**Comment by l5io** [comment]
🚀 Preview for commit e62009b87fa10481f620135cdcd06a2fde611e2e at: https://671a8c12c6e19e1d5fbe7f33--layer5.netlify.app

File: src/collections/blog/2024/09-27-meet-the-maintainer-aisuko-li/aisuko-li-layer5-maintainer.png
Comment (leecalcote) [inline src/collections/blog/2024/09-27-meet-the-maintainer-aisuko-li/aisuko-li-layer5-maintainer.png:1]: We have a newer, cooler member card style (in Figma) that can be used, if you like.

File: src/collections/blog/2024/09-27-meet-the-maintainer-aisuko-li/post.mdx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+    <br />
+    The GNU Hurd project is unique, so there’s no need to compare it with others.
+    In the Layer5 community, we have a warm welcome for new contributors, and most
+    projects have active reviewers who provide feedback quickly.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Anita:</span>
+  <p>What is so fascinating about service meshes?</p>
```
Comment (leecalcote) [inline src/collections/blog/2024/09-27-meet-the-maintainer-aisuko-li/post.mdx:117]: @hargunkaur286 terrible question. I've repeatedly asked that we distance the project from service meshes as much as possible. Will you please see that this question is either removed or replaced with another?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@hargunkaur286 you are most welcome to insert your name as the interviewer, if you like.

**Comment by l5io** [comment]
🚀 Preview for commit 897b1c349324f8e528f8e9e5466f105391df9ede at: https://6725b749667d6a1cfc6b39a8--layer5.netlify.app

**Pull request merged**